starter_project/autonomy/src/perception.cpp

```diff
@@ -0,0 +1,85 @@
+#include "perception.hpp"
+
+// ROS Headers, ros namespace
+#include <ros/init.h>
+
+int main(int argc, char** argv) {
+    ros::init(argc, argv, "starter_project_perception"); // Our node name (See: http://wiki.ros.org/Nodes)
+
+    [[maybe_unused]] mrover::Perception perception;
+
+    // "spin" blocks until our node dies
+    // It listens for new messages and calls our subscribed functions with them
+    ros::spin();
+
+    return EXIT_SUCCESS;
+}
+
+namespace mrover {
+
+    Perception::Perception() : mNodeHandle{}, mImageTransport{mNodeHandle} {
+        // Subscribe to camera image messages
+        // Every time another node publishes to this topic we will be notified
+        // Specifically the callback we passed will be invoked
+        mImageSubscriber = mImageTransport.subscribe("camera/color/image_raw", 1, &Perception::imageCallback, this);
```

Comment:
it worked for me with camera/left/image as well!

Reply:
FYI this is because on master @jmaff added a cool feature to simulate a stereo camera. Either should work as they are relatively close to the center of the rover.